crates/iota/src/genesis_ceremony.rs

```diff
@@ -118,6 +118,12 @@ pub enum CeremonyCommand {
         #[clap(long, name = "iota|<full-url>", help = "Remote migration snapshots.")]
         #[arg(num_args(0..))]
         remote_migration_snapshots: Vec<SnapshotUrl>,
+        #[clap(
+            long,
+            help = "Path to the delegator map file.",
+            name = "delegator_map.csv"
+        )]
+        delegator_map: Option<PathBuf>,
```

Comment:
Can you create a dedicated command similar to `InitTokenDistributionSchedule`?

Reply:
done: `InitDelegatorMap`